### DISCUSSION TITLE
SEB Locked

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-win-refactoring#772] [Q&A]
Hi, I had a quick question. What could be the cause of this error? Note that the SEB config file being used does not have a quit/config password. All it has is an administrator password (which is what we want).

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/e0b2b13e-2ad6-4476-afb4-f75dddd75658)
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/d356aa53-195c-4e1c-9d14-1f1a1ee41409)

Why is it asking me for a password? Also, what could have caused this error to pop up?

**Comment by dbuechel** [comment]
> Why is it asking me for a password?

Because the active user had changed or the computer had been locked, i.e. someone tried to log on to another user account or locked the currently active user session. More details can be found in the client application log, located at `%LocalAppData%\SafeExamBrowser\Logs`.

**Reply by dbuechel** [reply]
If there is no quit/unlock password set, you can simply unlock the lock screen with the respective button. Please take note of #305.
